glustercli/cmd/volume-create.go

```diff
@@ -16,20 +17,35 @@ const (
 )
 
 var (
-	flagCreateStripeCount                                                           int
-	flagCreateReplicaCount, flagCreateArbiterCount                                  int
-	flagCreateDisperseCount, flagCreateDisperseDataCount, flagCreateRedundancyCount int
-	flagCreateTransport                                                             string
-	flagCreateForce                                                                 bool
-	flagCreateAdvOpts, flagCreateExpOpts, flagCreateDepOpts                         bool
-	flagCreateThinArbiter                                                           string
-	flagCreateVolumeOptions                                                         []string
+	flagCreateStripeCount             int
+	flagCreateReplicaCount            int
+	flagCreateArbiterCount            int
+	flagCreateDisperseCount           int
+	flagCreateDisperseDataCount       int
+	flagCreateDisperseRedundancyCount int
+	flagCreateTransport               string
+	flagCreateForce                   bool
+	flagCreateAdvOpts                 bool
+	flagCreateExpOpts                 bool
+	flagCreateDepOpts                 bool
+	flagCreateThinArbiter             string
+	flagCreateVolumeOptions           []string
+
+	flagCreateVolumeSize            string
+	flagCreateDistributeCount       int
+	flagCreateLimitPeers            string
+	flagCreateLimitZones            string
+	flagCreateExcludePeers          string
+	flagCreateExcludeZones          string
+	flagCreateSnapshotEnabled       bool
+	flagCreateSnapshotReserveFactor float64 = 1
+	flagCreateSubvolZoneOverlap     bool
```

Comment:
I'm wondering if instead of dumping all these flags as global variables across the package, we could do something better.
Create a flag struct for each command, and have the individual flags as members, and have single variable of this struct for each command.

Like this for eg.,
```go
type volCreateFlags struct {
  stripeCount, replicaCount, arbiterCount int
  // other counts
  force, advOpts, expOpts, depOpts bool
 // other flags
}

var createFlags volCreateFlags

volumeCreateCommand.Flags().IntVar(&createFlags.replicaCount, "replica", 0, "Replication count")
```